### PR TITLE
set flywheel to coast if no note held

### PR DESCRIPTION
**Pull request by outsidermm**

### CODE REVIEW

File: components/shooter.py [hunk 1]
```diff
@@ -110,6 +110,8 @@ def __init__(self) -> None:
         self.inclinator_controller.setTolerance(ShooterComponent.INCLINATOR_TOLERANCE)
         SmartDashboard.putData(self.inclinator_controller)
 
+        self.desireStop = False
```
Comment (auscompgeek) [inline components/shooter.py:113]: nit: [use `snake_case` for variable names](https://wiki.thedropbears.org.au/sections/software/coding-standards#h.p_1tiHT35Sqjyr)

```suggestion
        self.desire_stop = False
```

File: components/shooter.py [hunk 2]
```diff
@@ -165,6 +167,9 @@ def set_range(self, range: float) -> None:
             )
         )
 
+    def set_stop(self) -> None:
```
Comment (auscompgeek) [inline components/shooter.py:170]: I think `stop` is a verb here

```suggestion
    def stop(self) -> None:
```

**Review by auscompgeek — APPROVED** [review]
LGTM if LGT @mlists

**Review by mlists — APPROVED** [review]
Pls squash merge

**Pull request merged**